src/main/webapp/itempagescript.js

```diff
@@ -2,7 +2,7 @@
  * Retrieves ItemPageData and forms page using other functions.
  */
 async function loadItemPage() {
-  const itemId = getUrlParam('itemId');
+  const itemId = getItemId();
```

Comment:
This should be using `getUrlParam('itemId')` instead of `getItemId()`.
Bryan added the new `getUrlParam` method.

Same for line 46.

Reply:
There isn't any getUrlParam function in this file so it doesn't run.

Reply:
I think its because you removed `<script src="script.js"></script>` in your item-page.html file. Can you re-add that and try it?

See Bryan's PR which added the function `getUrlParam()`: https://github.com/googleinterns/step110-2020/pull/31/files